src/preact.d.ts

```diff
@@ -277,8 +284,8 @@ declare namespace JSX {
 		charSet?:string;
 		challenge?:string;
 		checked?:boolean;
-		class?:string;
-		className?:string;
+		class?:string | Object;
+		className?:string | { [key:string]: string | boolean | number };
```

Comment:
ah, then it should only allow boolean right? string and number are unnecessary
and it should be the same for `class` and `className` right?